scanapp/models.py

```diff
@@ -42,6 +42,7 @@ class Scan(models.Model):
     """
     Store various attributes of a scan
     """
+
```

Comment:
What is this extra line about?

Reply:
`Method definitions inside a class are surrounded by a single blank line.`
This is a point in pep8 guideline, I guess that's why coala added a new line while re-formatting.

Reply:
Ya, there should be a blank line above methods, like `__str__`.
This rule can be disabled if scancode maintainers prefer to not have a space here.